### PR TITLE
Implemented a cli table formatting feature

### PR DESCRIPTION
**Pull request by Nick-S-2018**
Added response formatting for the 'cli' endpoint; changed default Manticore client endpoint to `Sql`, updated TestFunctionalTrait and tests themselves to consider this; added base executor classes for Manticore queries and formattable queries, updated ShowFullTables executor to support cli formatting; added unit and functional tests for the `Cli table` functionality

### CODE REVIEW

File: src/Lib/TableFormatter.php
Comment (donhardman) [inline src/Lib/TableFormatter.php:73]: It's preferred to use `array_map` here
`$table .= implode(PHP_EOL, array_map($this->row(...), $data)) . PHP_EOL;`

File: src/Lib/TableFormatter.php
Comment (donhardman) [inline src/Lib/TableFormatter.php:78]: Due to the table being a string, why not make it default as an empty string and get rid of nullable? So simply if the string

File: src/Network/ManticoreClient/HTTPClient.php
Comment (donhardman) [inline src/Network/ManticoreClient/HTTPClient.php:101]: Can you explain this magic?

File: src/Network/Response.php
Comment (donhardman) [inline src/Network/Response.php:77]: Do we really this line?

**Review by donhardman — CHANGES_REQUESTED** [review]
I've wrote some questions and also issues that found, please check and reply

File: src/ShowFullTables/Executor.php
Comment (Nick-S-2018) [reply src/ShowFullTables/Executor.php:42]: You mean in the request class?  Yes, it will be better.   I've made this update here first, during testing, and then forgot about this part )

File: src/ShowFullTables/Executor.php
Comment (Nick-S-2018) [reply src/ShowFullTables/Executor.php:53]: Maybe, we can define some struct for the Task's output array, then?  You seem to have mentioned about it too. 
E.g.  `array{'result': actual_result}`  Where `actual_result` can be of various types.

File: src/ShowQueries/Executor.php
Comment (Nick-S-2018) [reply src/ShowQueries/Executor.php:144]: The idea is to create a parent class (ClientQueryExecutor) comprising all the executors that need manticoreClient.

File: src/Base/ClientQueryExecutor.php
Comment (Nick-S-2018) [reply src/Base/ClientQueryExecutor.php:39]: Now, if there's a command that doesn't need it ( like Backup ), its executor just implements CommandExecutorInterface as before, not inheriting from ClientQueryExecutor.

File: src/Base/FormattableClientQueryExecutor.php
Comment (Nick-S-2018) [reply src/Base/FormattableClientQueryExecutor.php:20]: Yes, now InsertQuery executor doesn't need this, so I've decided to add this specific parent class, just for the command that do.

File: src/Network/ManticoreClient/HTTPClient.php
Comment (Nick-S-2018) [reply src/Network/ManticoreClient/HTTPClient.php:101]: Yes, remember, I've discussed this with Stas in the chat. The purpose is that daemon would still be able to redirect some error requests from Buddy back to it. ( That's needed  for the processing of error `/cli` requests).  To let daemon do this, we must get off the Buddy's agent header.

File: src/ShowFullTables/Executor.php
Comment (donhardman) [reply src/ShowFullTables/Executor.php:42]: Yeah, I guess just it's better to get rid of confusing reassigment like this :)

File: src/ShowFullTables/Executor.php
Comment (donhardman) [reply src/ShowFullTables/Executor.php:53]: I think it's a good idea to create Struct (Class) that will be converted into an array for a response. So we can follow strict types in that way.

File: src/ShowQueries/Executor.php
Comment (donhardman) [reply src/ShowQueries/Executor.php:144]: OK, noted. Missed it first time :)

File: src/Network/ManticoreClient/HTTPClient.php
Comment (donhardman) [reply src/Network/ManticoreClient/HTTPClient.php:101]: I did not get the case. Can you explain some examples of when we will get into this issue? Just to understand if we can find better solution for this.

**Review by donhardman — APPROVED** [review]
Looks good now. But still, need to get working manticore in our docker image to make CI works. Once it's ready, we can merge. But as I know, for now, we are blocked by the flow deciding issues for future releases for now.

**Pull request merged**
